### ISSUE TITLE
 Please confirm what is being sent to server re: paper advancing

### ISSUE THREAD

**Issue by akshayvernekar** [akshayvernekar/Brilliant_pad_issue_management#12]
Four ways to Advance Sheet:
- User presses and holds (and later releases) Pad Advance button on machine?
- User double clicks Pad Advance button on machine to advance a full sheet?
- User sends Advance Sheet command from App?
- App or Server advance sheet according to auto mode or timer mode?

Three ways to Stop a Sheet:
- User stops an advancing sheet by pressing button on machine
- User stops an advancing sheet by pressing “Clear Error” on app?
- Machine stops an advancing sheet due to an error (out of paper, motor current, other)

AKSHAY: Does count info to server work same way for each method of advancing sheet described above?

AKSHAY AND PRAVEENA: Also consider these situations:
• What happens when user presses”Advance Sheet” while there is detection?
• What happens when firmware update = successful?
• What happens when firmware update = non successful?

**Comment by akshayvernekar** [comment]
Hi Alan, The pad count is reported every time a advance is complete be it auto,timer or manual(ie button press) . Status is sent in below format .

> "{
        ""ts"":        1563287808,
        ""type"":        ""PAD_ADVANCE"",
        ""value"":        {
                ""source"":        ""auto"",
                ""mark_count"":        1
        }
}
"

**Comment by alanjcook** [comment]
Please ensure log displays pad advances and marks advanced so we can confirm this.

**Comment by akshayvernekar** [comment]
We already log in the device console. You can ask @Praveena to display this .

**Comment by cumulations** [comment]
For the advance pad command, we get the remaining mark count in the response.
`{
        "ts":        1563287808,
        "type":        "PAD_ADVANCE",
        "value":        {
                "source":        "auto",
                "mark_count":        1
        }
}
` 
We can display Pad advance event in the event logs and also the remaining pad count. Would this be ok ?

**Comment by alanjcook** [comment]
Yes. See comment in #19: When paper is advanced --  put "paper advanced x marks. x marks remaining" into log.
